org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -601,6 +610,28 @@ SsoApplicationManager:
         ]
       }
 
+# Role for a user that can access BedRock in the Synapse LLM prod account
+SSoLLmDeveloper:
```

Comment:
is this specific to synapse?  if so maybe it should be `SynapseSsoLlmDeveloper`?

Reply:
Will check.